cmd/test/main.go

```diff
@@ -287,9 +282,6 @@ func validateK8sVersionFormatIfProvided(k8sVersion string) error {
 func Test(ctx *TestCommandContext, paths []string, prerunData *TestCommandData) error {
 
 	if paths[0] == "-" {
-		if len(paths) > 1 {
```

Comment:
If we move this check to `Args` then it is not tested in the kustomize call anymore https://github.com/datreeio/datree/blob/a403a0cbe12883efcd855709123799be442200f2/cmd/kustomize/main.go#L87

I think we want to keep this check here

Reply:
IMO, it would be better to introduce `Args` block in `Kustomize`. I'm expecting going forward we will gonna introduce it. Also, it will give more control individually over each command `test` and `kustomize`.

WDYT?